components/_patterns/01-atoms/06-buttons/_buttons.scss

```diff
@@ -19,32 +24,32 @@
   }
 }
 
-@mixin button-alt {
-  @include button($gray-medium, $white, $black);
-
-  font-weight: 600;
-  font-size: 0.7rem;
-  padding: 0.4em 0.6em 0.3em;
-}
+@mixin button--outline {
+  background-color: $white;
+  border: 1px solid $gray-light;
+  border-radius: 2px;
```

Comment:
The only place we've used this so far is in the buttons. And there are other border-radiuses for the round buttons, so I don't think it could just be a variable like `button-border-radius: 2px`. But there were a number of shared styles between the two buttons we have, so I pulled all of those into a mixin.

We'll probably adjust this as we create more button styles, and other components, but for now, I think this is good.